packages/salesforcedx-vscode-core/src/messages/i18n.ts

```diff
@@ -250,7 +250,10 @@ export const messages = {
   demo_mode_prompt:
     'Authorizing a business or production org is not recommended on a demo or shared machine. If you continue with the authentication, be sure to run "SFDX: Log Out from All Authorized Orgs" when you\'re done using this org.',
   force_auth_logout_all_text: 'SFDX: Log Out from All Authorized Orgs',
+  manifest_editor_dupe_error: 'Manifest with the name %s already exists. Delete this manifest or use another name.',
   force_auth_logout_default_text: 'SFDX: Log Out from Default Org',
+  manifest_editor_save_placeholder: 'Enter unique manifest name (no .xml)',
```

Comment:
This seems like it should be `package` since that's what we default to if there's not text inputed. We tell them if the validation fails so I think it's fine to not call out that `.xml` isn't allowed until they try to enter it.

Reply:
Feedback from @sbudhirajadoc on the string if we want a heavier hand here: `Enter unique a manifest file name (without file extension)`